src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -569,6 +569,16 @@ type VehicleRentalStation implements Node & PlaceInterface {
     """
     spacesAvailable: Int
 
+    """
+    Number of vehicles currently available on the rental station, grouped by vehicle type.
+    """
+    vehicleTypesAvailable: [RentalVehicleTypeCount!]
+
+    """
+    Number of free spaces currently available on the rental station, grouped by vehicle type.
+    """
+    vehicleSpacesAvailable: [RentalVehicleTypeCount!]
```

Comment:
Just thinking out aloud: whenever I read a repeated word in the schema I wonder if these two properties should be grouped together. Perhaps something like this:

```
availability {
  types: VehicleTypeCount
  spaces: VehicleTypeCount
}
```

cc @optionsome @t2gran 

(Please don't do anything yet, @cedarbaum, as this thought is not fully formed yet. We will discuss in a dev meeting.)



cc @optionsome @t2gran

Reply:
@optionsome and I had a chat what we want the schema to look like and we have landed on the following(pseudo-graphqls language):

```
availableVehicles {
  total: Int
  byType: [VehicleTypeCount!]!
}

availableSpaces {
  total: Int
  byType: [VehicleTypeCount!]!
}
```
We also want to deprecate the `spacesAvailable` and `vehiclesAvailable` fields so it's not too confusing for the consumers.

Does this make sense, @cedarbaum ?

Reply:
This makes sense, I went ahead and updated the PR with the new query structure. I did keep the `Rental` prefix for the GraphQL types to be consistent with other existing types, but let me know if you have a preference for these type names (`RentalVehicleEntityCounts` and `RentalVehicleTypeCount`, specifically).

I also noticed that @t2gran proposed an alternative query structure. Is there consensus with the maintainers about the final design or is it still being discussed? If it's still being discussed and it would be helpful for me to attend a future dev meeting, I am happy to do so!

Reply:
Well I thought there was a consensus until there wasn't. :)

We will definitely discuss this again today at our meeting at 3pm CET. Everybody is welcome to attend but it will not be required to get this merged.

Reply:
I am fine with the suggestion made by @optionsome and @leonardehrenfried.